sympy/polys/rootisolation.py

```diff
@@ -1742,7 +1742,7 @@ def __repr__(self):
     def is_disjoint(self, other):
         """Return ``True`` if two isolation intervals are disjoint. """
         if isinstance(other, RealInterval):
-            return (self.b <= other.a or other.b <= self.a)
+            return (self.b < other.a or other.b < self.a)
```

Comment:
There are two `is_disjoint` methods in this file, with multiple inequalities in both. Should they all be changed to strict inequalities?

Reply:
I noticed that, yeah. It seems like it's for complex intervals...

Reply:
The fix here seems to be correct. From what I can tell, all the inequalities in the two `is_disjoint` methods should be strict. Unless the intervals are are intended to be open intervals (?), in which case, something else is going wrong here.

Reply:
I went ahead and made all the inequalities strict. I ran `test_rootisolation` to make sure it didn't break anything, but the automated tests still have to run.